Source/WebKit/NetworkProcess/storage/OriginQuotaManager.h

```diff
@@ -50,11 +50,11 @@ class OriginQuotaManager : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPt
     void resetQuotaForTesting();
 
 private:
-    OriginQuotaManager(uint64_t quota, uint64_t standardReportedQuota, GetUsageFunction&&, IncreaseQuotaFunction&&, NotifyUsageUpdateFunction&&);
+    OriginQuotaManager(uint64_t quota, uint64_t standardReportedQuota, GetUsageFunction&&, IncreaseQuotaFunction&&, NotifySpaceGrantedFunction&&);
     void handleRequests();
     bool grantWithCurrentQuota(uint64_t spaceRequested);
     bool grantFastPath(uint64_t spaceRequested);
-    void usageUpdated(uint64_t newUsage);
+    void spaceGranted(uint64_t spaceRequested);
```

Comment:
The function definition uses amount and I tend to prefer amount instead of spaceRequested.